mmtrack/datasets/dataset_wrapers.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import random
+
+from mmdet.datasets.builder import DATASETS, build_dataset
+from torch.utils.data.dataset import ConcatDataset
+
+
+@DATASETS.register_module()
+class RandomSampleConcatDataset(ConcatDataset):
+    """A wrapper of concatenated dataset. Support randomly sampling one dataset
+    from concatenated datasets and then getting samples from the sampled
+    dataset.
+
+    Args:
+        dataset_cfgs (list[dict]): The list contains all configs of
+            concatenated datasets.
+        dataset_sampling_weights (list): The list contains the sampling
+            weights of each dataset.
+    """
+
+    def __init__(self, dataset_cfgs, dataset_sampling_weights=None):
```

Comment:
please check the legality of `dataset_sampling_weights`.
each element in `dataset_sampling_weights` should be bigger than 0 ( allow to equal to 0).